### PR TITLE
added reopen eval admin api

### PR DESCRIPTION
**Pull request by jatnerubia**
Ticket ID: [Create API to reopen Evaluation Administration](https://github.com/orgs/nerubia/projects/18/views/2?pane=issue&itemId=48745957)

### CODE REVIEW

**Review by chadiegilnerubia — APPROVED** [review]
:1st_place_medal:

File: src/services/evaluation-administration-service.ts [hunk 1]
```diff
@@ -344,6 +345,67 @@ export const publish = async (id: number) => {
...
+    const evaluationRatings = await EvaluationRatingRepository.aggregateSumByEvaluationId(
+      evaluation.id,
+      {
+        score: true,
+      }
+    )
+
+    const data = {
+      zscore: 0,
+      weighted_score: 0,
```
Comment (emmaacha) [inline src/services/evaluation-administration-service.ts:376]: not sure but should this be weighted_zscore too?

**Review by emmaacha — CHANGES_REQUESTED** [review]

File: src/services/evaluation-administration-service.ts (hunk 1 above)
Comment (jatnerubia) [reply src/services/evaluation-administration-service.ts]: @emmaacha done ^^

**Review by emmaacha — APPROVED** [review]
Cool

**Pull request merged**